tests/functionnal/Glpi/ContentTemplates/Parameters/ChangeParameters.php

```diff
@@ -85,20 +85,7 @@ public function testGetValues(): void
                 'completename' => 'category_testGetValues',
             ],
             'requesters' => [
-                'users'  => [
-                    [
-                        'id'       => getItemByTypeName("User", TU_USER, true),
-                        'login'     => TU_USER,
-                        'fullname' => TU_USER,
-                        'email'    => "_test_user@glpi.com",
-                        'phone'    => null,
-                        'phone2'   => null,
-                        'mobile'   => null,
-                        'firstname'  => null,
-                        'realname'   => null,
-                        'used_items' => [],
-                    ],
-                ],
```

Comment:
Maybe it would be better to properly set the requester in the code above (`$this->createItem('Change', [...`) rather than removing him from the expected data ?

Reply:
not yet.
But you're right, i need to check also this behavior, you mentioned it yesterday.

Reply:
Please check #10950 for this issue.
If you agree the current pr can be merged.